core/src/main/java/com/amplifyframework/predictions/models/FaceLivenessSession.kt

```diff
@@ -16,14 +16,13 @@
 package com.amplifyframework.predictions.models
 
 import com.amplifyframework.annotations.InternalAmplifyApi
-import com.amplifyframework.core.Action
 
 @InternalAmplifyApi
 class FaceLivenessSession(
     val challenges: List<FaceLivenessSessionChallenge>,
     private val onVideoEvent: (VideoEvent) -> Unit,
     private val onChallengeResponseEvent: (ChallengeResponseEvent) -> Unit,
-    private val stopLivenessSession: Action
+    private val stopLivenessSession: (Int?) -> Unit
```

Comment:
This check would break ui unit tests if we ran the tests against the latest version of Amplify

Reply:
unit tests have been fixed in the [UI PR](https://github.com/aws-amplify/amplify-ui-android/pull/106)